fedora-vbx.md

```diff
@@ -0,0 +1,74 @@
+### Install Vagrant using VirtualBox on Fedora 24
+
+Tested for Fedora 24
+
+Run the following commands on fresh installed Fedora first:
+```shell
+# Upgrade fedora
+sudo dnf -y update          # upgrade to the latest kernel etc.
+
+# Install vim editor
+sudo dnf -y install vim
+
+# Reboot machine
+sudo reboot
+```
+
+Have you run `dnf update` lately, then start here:
+```shell
+# Generate SSH key to use when accessing machines
+ssh-keygen -t rsa -b 4096 -N "" -C "root@example.com" -f ~/.ssh/rhce # empty password
+
+# Make loading SSH key to ssh agent automatic
+eval $(ssh-agent); echo $'eval $(ssh-agent)' >> ~/.bash_profile
+ssh-add ~/.ssh/rhce; echo $'ssh-add ~/.ssh/rhce' >> ~/.bash_profile
+
+# Download VritualBox repository
+sudo wget http://download.virtualbox.org/virtualbox/rpm/fedora/virtualbox.repo -O /etc/yum.repos.d/virtualbox.repo
+
+# Clean repository cache & import GPG Key
+sudo dnf clean dbcache
+sudo dnf -y repolist        # Import the GPG keys
+
+# Get vagrant version
+export virtualbox_version=`yum -y search VirtualBox | awk 'match($0, /(VirtualBox-[0-9]\.[0-9])/, a) {b=a[1]} END {print b}'`  # depends on yum sorting
+export virtualbox_version=`yum -y search VirtualBox | awk 'match($0, /(VirtualBox-[0-9]\.[0-9])/, a)  {if (RSTART)  { ++i; v[i]=a[1]}} END {n = asort (v); print v[n]}'`
```

Comment:
I was [studying](http://shop.oreilly.com/product/9781565922259.do) regex, `sed` & `awk` and came up with several alternatives. [The other`.md` files](https://github.com/AnwarYagoub/RHCSA-RHCE-Lab-Environment/tree/update_readme) have similar pairs and in [c6bcfb3](https://github.com/AnwarYagoub/RHCSA-RHCE-Lab-Environment/pull/20/commits/c6bcfb3c904e137c541954fa5b721f3150a3a096) you'll find even more alternatives. 

What do you think, should we depend on the output order of `yum` & html pages, or should `awk` do the sorting?  Any other suggestions, enhancements, questions, etc?  

[Ubuntu.md](https://github.com/AnwarYagoub/RHCSA-RHCE-Lab-Environment/blob/update_readme/ubuntu.md) currently has three steps for installing vagrant:
1. download the `.deb` file 
1. install it `apt install ./vagrant_"$vagrant_version"_x86_64.deb`
1. delete it `rm ./vagrant_"$vagrant_version"_x86_64.deb`

whereas in CentOS/Fedora it is a simple `yum install https://...._x86_64.rpm`. Do you know how to put the `.deb`-url on the `apt install` command-line directly?